src/sentry/models/project.py

```diff
@@ -249,12 +249,19 @@ def get_member_alert_settings(self, user_option):
         :param user_option: alert option key, typically 'mail:alert'
         :return: A dictionary in format {<user_id>: <int_alert_value>}
         """
-        from sentry.models import UserOption
+        from sentry.models import UserOption, User
+
+        member_alert_disabled = {}
+        for user in User.objects.get_from_projects(self.organization.id, [self]):
+            is_disabled = {
+                o.user_id: int(o.value)
+                for o in UserOption.objects.filter(user=user)
```

Comment:
Should this still be filtered by `key`?

Reply:
Maybe there's another solution, but when I was filtering by `key`, it was returning the wrong `value` if the user Disabled all alert notifications (see other comment).